nuxeo-core-binarymanager-s3/src/test/java/org/nuxeo/ecm/core/storage/sql/TestS3BinaryManager.java

```diff
@@ -163,6 +163,27 @@ protected void doTestS3MaxConnections() throws Exception {
         o.close();
     }
 
+    @Override
+    @Test
+    public void testStoreFile() throws Exception {
+        // Run normal test
+        super.testStoreFile();
+        // Run corruption test
+        String key = binaryManager.bucketNamePrefix + CONTENT_MD5;
+        binaryManager.amazonS3.putObject(binaryManager.bucketName, key, "Georges Abitbol");
+        binaryManager.fileCache.clear();
+        Boolean exceptionOccured = false;
```

Comment:
Can be lowercase `boolean`

Reply:
I know but as this is a backport I'd rather keep the code identical with what's in master.